src/nora/components/NoraTextAreaInput/NoraTextAreaInputExample.module.scss

```diff
@@ -0,0 +1,13 @@
+.LabelClassExample {
+  font-weight: 500;
+  font-size: 11px;
```

Comment:
Should we add a Nora small font size? I feel like I've been seeing a few of these 11px